e2e/errors.go

```diff
@@ -27,6 +27,7 @@ func isRetryableAPIError(err error) bool {
 		apierrors.IsTooManyRequests(err) || utilnet.IsProbableEOF(err) || utilnet.IsConnectionReset(err) {
 		return true
 	}
+
```

Comment:
Maybe we can remove this extra space if not required.

Reply:
This is kept for block separation. IMO its not a problem

Reply:
okay.